index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <h1 id = "title"> Survey Form </h1>
+  </head>
+  <body>
+<p id = "description"> Description </p>
+   
+<form id ="survey-form"> </form>
+
+<div id = "survey-form">
+<label id="name-label"><b>Name:</b></label>
+            <input id="name" type="name" name="name" class="form" required placeholder="Full Name"><br>
+            <br>
+            </div>
+
+            <div id ="survey-form">
+            <label id="email-label"><b>Email:</b></label>
+            <input id="email" type="email" name="email" class="form" required placeholder="example@mail.com"><br>
+            <br>
+             </div>
```

Comment:
Great job creating a form attribute but I can see that the the form components are not in the form attribute. Kindly work on this.

Reply:
So awesome, I like your comments